person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_accessor :name, :age
+
+  def inizialiate(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+
+  def can_use_services?
+    return true if is_of_age? && @parent_permission
```

Comment:
- [ ] Great job so far 👏🏻 every is fine but According to the project requirements, the public method `can_use_services?` should return true if the person is of age `or` has permission from parents. Kindly use the "or" sign `||` and not the "and" signed `&&` It should be like this:

```ruby
  def can_use_services? 
    is_of_age?() || @parent_permission
  end
```

Reply:
thank you!